frontend/src/metabase/admin/app/reducers.ts

```diff
@@ -47,13 +47,13 @@ export const getAdminPaths: () => AdminPath[] = () => {
       path: "/admin/tools",
       key: "tools",
     },
-  ];
-
-  items.push(...PLUGIN_ADMIN_NAV_ITEMS, {
+    ...PLUGIN_METABOT.adminNavItem,
```

Comment:
PLUGIN_ADMIN_NAV_ITEMS was not actually used, I think it became permanently empty after we removed audit. I removed the plugin altogether in favor of pushing in specific named items from specific plugins so that it's clear what might actually be there.